<launch>
    <arg name="look_ahead" default="1.00"/>
    <arg name="distance_threshold" default="0.25"/>
    <arg name="path_follow_rate" default="20.0" />
    <arg name="use_avoidance" default="false" />
    <arg name="loiter_duration" default="0.0"/>
    <arg name="self_start" default="true" />

    <!-- Launch the Path Follower, which will arm the drone and get ready to follow paths -->
    <node name="drone_local_planner" pkg="drone_navigation" type="drone_path_follower"  output="log">
        <param name="look_ahead" type="double" value="$(arg look_ahead)"/>
        <param name="distance_threshold" type="double" value="$(arg distance_threshold)"/>
        <param name="path_follow_rate" type="double" value="$(arg path_follow_rate)"/>
        <param name="use_avoidance" type="bool" value="$(arg use_avoidance)"/>
        <param name="loiter_duration" type="double" value="$(arg loiter_duration)"/>
        <param name="self_start" type="bool" value="$(arg self_start)"/>
    </node>
    
</launch>

